resources/css/bem/beatmap-discussion-post.less

```diff
@@ -30,6 +30,10 @@
   }
 
   &--new-reply {
+    // use padding because this section is a different background colour to the parent
+    // and fortunately doesn't need an anchor.
+    margin: 0;
+    padding: 10px 0;
```

Comment:
The bottom padding isn't supposed to be that big. Also the close button alignment seems wrong?

![](https://s.myconan.tk/2023/09/firefox_2023-09-12_19-55-53.png)

this is how it previously looked like
![](https://s.myconan.tk/2023/09/firefox_2023-09-12_19-59-19.png)